Project-II-Calculator/calculator/src/components/DisplayComponents/Display.scss

```diff
@@ -0,0 +1,10 @@
+.display{
+    font-size: 40px;
+    background-color: darkgrey;
+    height: 67px;
```

Comment:
Minor detail, but why change the `background-color` of the display area from a blackish color (as per the design file) to `darkgrey`?